pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/patch/BasePatch.java

```diff
@@ -116,6 +128,7 @@ Optional<Object> processSingleNameValue() {
 
     List<Object> processMultipleNameValues() {
         List<Object> updatedNameValues = new ArrayList<>();
+        exportToESWithId(patch.getId(), patch.getNameValues());
```

Comment:
can we change the order here please. I would like to first write into db and then into ES.